src/components/NavBar/index.tsx

```diff
@@ -39,7 +39,7 @@ const NavBar = function ({ UserAvatar }: Props) {
       </a>
       <a
         className={`${navBarStyles.navItem} menu`}
-        href="https://hyperplay.xyz/developers"
+        href="https://www.hyperplay.xyz/developers"
```

Comment:
this should be a separate PR so it doesn't get blocked by other changes you are introducing here